### ISSUE TITLE
1.1.0, 1.1.1 - error when starting the pooled timer

### ISSUE THREAD

**Issue by bubenkoff** [headzoo/react-moment#129]
Uncaught TypeError: Super expression must either be null or a function

Getting this error in production build only, for some reason

**Comment by headzoo** [comment]
Is there any kind of stacktrace? Any differences between dev and production, like different operating systems?

**Comment by bubenkoff** [comment]
as it's a production build, there's not much known unfortunately
it's a browser app, no difference in browsers prod vs dev
do you run the recent version in production with no problem?

**Comment by maoryadin** [comment]
Same for me, happen only in production.
something related to react-moment.

![typeError](https://user-images.githubusercontent.com/31710467/104836224-19328c00-58b5-11eb-935f-b7e0f8dcad0b.png)

**Comment by maoryadin** [comment]
Solved by downgrade to version 0.9.7

@bubenkoff

**Comment by bubenkoff** [comment]
@maoryadin yes, I also 'solved' it like this

**Comment by headzoo** [comment]
Are either of you using UglifyJS in your webpack config? Adding `keep_fnames: true` to your UglifyJS config might fix the problem.

**Comment by bubenkoff** [comment]
@headzoo yes I do, but keeping ALL the fnames will increase the bundle quite a lot, which function name exactly should be preserved?

**Comment by martinvergara26** [comment]
Same problem here

**Comment by headzoo** [comment]
Closing old open issue during cleanup.
